### ISSUE TITLE
Help with getting the desired results

### ISSUE THREAD

**Issue by alonfnt** [sbakker/Adwaita-Steven#1]
Hi, I'm having troubles achiving the following look

![adwaita-dark-headerbars](https://user-images.githubusercontent.com/38870744/81870705-bc890f80-9576-11ea-9cab-cbdd4eb5abd8.png)

Could  you indicate me which values should I change to achieve it? Thanks

**Comment by sbakker** [comment]
Hi Albert,

I'm not sure I understand your problem. Is the above screenshot the current state? If so, what is the desired end state?

In case the question is how to set up/install the theme, then I understand the README is little light on those details. Let me know if this is the problem you have, and I'll add some more information on how to install and select the theme.

**Comment by alonfnt** [comment]
Sorry I realized I was very vague on my issue! I wanted to achieve the screenshot look. However after digging more I saw that it wasn't as trivial as I thought, since there are a lot of tweaks to be made at the _commons file and I am sure your script wasn't thought with this issue in mind.

Anyway I gave up and if I find the time I'll try to go deep on it.
I close this issue but feel free to recommend an easier method to achieve it if you know it. 
Thank you!
